testing/camunda-process-test-java/src/main/java/io/camunda/process/test/api/CamundaProcessTestExtension.java

```diff
@@ -213,8 +213,9 @@ public void afterEach(final ExtensionContext extensionContext) {
     }
     CamundaAssert.reset();
     closeCreatedClients();
-    // final step: delete data
+    // final steps: delete data and reset the time
     deleteRuntimeData();
+    resetRuntimeClock();
```

Comment:
:thought_balloon: We also need to reset the clock in the Spring execution listener.

Reply:
:thought_balloon: We should write a test case to verify that the clock is reset.